eng/tools/versioning/set-dev.js

```diff
@@ -179,7 +179,8 @@ async function main(argv) {
   for (const package of Object.keys(rushPackages)) {
     if (
       ["client", "core"].includes(rushPackages[package].versionPolicy) &&
-      rushPackages[package].projectFolder.startsWith(`sdk/${service}`)
+      rushPackages[package].projectFolder.startsWith(`sdk/${service}`) &&
+      !rushPackages[package].json["private"]
```

Comment:
Does this really help with anything?

Reply:
yes. it won't change attach the dev versioning for private packages

Reply:
this will work for all private packages even if they forget to set the skippublishdevfeed variable

Reply:
But all it does is prevent the dev version from being set on the package it doesn't do anything about not publishing it. While I don't mind this change I'm not sure it really adds much for us.